Description: Relax dependency on libnetfilter_conntrack 1.0.2

Debian wheezy does not carry libnetfilter_conntrack >= 1.0.2, only 1.0.1

--- ulogd2-2.0.2.orig/configure.ac
+++ ulogd2-2.0.2/configure.ac
@@ -60,7 +60,7 @@ AM_CONDITIONAL([BUILD_NFLOG], [test "x$e
 AC_ARG_ENABLE(nfct,
        AS_HELP_STRING([--enable-nfct], [Enable nfct module [default=yes]]),,[enable_nfct=yes])
 AS_IF([test "x$enable_nfct" = "xyes"], [
-    PKG_CHECK_MODULES([LIBNETFILTER_CONNTRACK], [libnetfilter_conntrack >= 1.0.2])
+    PKG_CHECK_MODULES([LIBNETFILTER_CONNTRACK], [libnetfilter_conntrack >= 1.0.1])
     AC_DEFINE([BUILD_NFCT], [1], [Building nfct module])
 ])
 AM_CONDITIONAL([BUILD_NFCT], [test "x$enable_nfct" = "xyes"])
